.github/CHANGELOG.md

```diff
@@ -50,6 +64,29 @@
     Hamiltonians, respectively, and apply the corresponding QAOA cost and mixer layers
     to the quantum circuit
 
+  For example, using PennyLane to construct and solve a MaxCut problem with QAOA:
+
+  ```python
+  wires = range(3)
+  graph = Graph([(0, 1), (1, 2), (2, 0)])
+  cost_h, mixer_h = qaoa.maxcut(graph)
+
+  def qaoa_layer(gamma, alpha):
+      qaoa.cost_layer(gamma, cost_h)
+      qaoa.mixer_layer(alpha, mixer_h)
+
+  def antatz(params, **kwargs):
+
+      for w in wires:
+          qml.Hadamard(wires=w)
+
+      # layer the QAOA ansatz 2 times
```

Comment:
```suggestion
      # repeat the QAOA layer two times
```
:grinning:

Reply:
lol I know it sounded awkward, but I was deliberately trying to use 'layer' as a verb as `qml.repeat` vs. `qml.layer` is still fresh in my head.

`qml.layer` in the line below sounded to me like you're _making_ a layer, not that you are 'layering' an existing layer